### PR TITLE
Book navigation

### PR DESCRIPTION
**Pull request by Jason-Lopez8**
Created a "SelectList" similar to SelectGrid.

Got relatively far, hit a roadblock with the "Book, Chapter, and Verse" tabs at the top. They're in the same list format as the book collections.

### CODE REVIEW

**Comment by chrisvire** [comment]
@Jason-Lopez8 I like the list display.  However, you code is how of date with main and doesn't have the changes for switching themes and the SelectList isn't themed correctly.

You will need to rebase your code (get with me or @jakedcolbert on how to do this). Then look at @jakedcolbert PR on dropdown styles (and/or ask him for help) on theming the select list.

Also, the dropdown arrow next to the book name is not colored correctly.

File: src/lib/components/SelectList.svelte [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    import config from '$lib/data/config';
+    export let options = [''];
+
+    const dispatch = createEventDispatcher();
+    $: rows = Math.ceil(options.length);
+    const colors = (type: string, key: string) =>
+        config.themes.find((x) => x.name === 'Normal').colorSets.find((x) => x.type === type)
+            ?.colors[key];
+    const textColor = colors('main', 'ChapterButtonTextColor');
+    let tableColor = colors('main', 'BackgroundColor');
```
Comment (jakedcolbert) [inline src/lib/components/SelectList.svelte:16]: Use the $s store to retrieve colors. I believe everything you need is in under the ui.select-list-button key.

**Review by jakedcolbert — CHANGES_REQUESTED** [review]

**Comment by chrisvire** [comment]
@Jason-Lopez8 I have merged @jakedcolbert's PR on Dropdown Style.  You should rebase this PR on main and use what he suggests.  If you need help with the rebase, let me know.

**Comment by chrisvire** [comment]
@Jason-Lopez8 For the List, you need to use book names instead of the book abbreviations.

Anything else you are waiting on doing?

**Review by chrisvire — APPROVED** [review]

**Pull request merged**